app/helpers/advice_page_helper.rb

```diff
@@ -55,6 +55,19 @@ def format_rating(rating)
     rating > 4 ? "Limited variation" : "Large variation"
   end
 
+  def compare_benchmark_key_for(advice_page_key)
```

Comment:
Minor: but if this is for the school groups page, then maybe the helper should be in the SchoolGroupsHelper?